Controller/CRUDController.php

```diff
@@ -265,6 +265,9 @@ public function editAction($id = null)
 
             // persist if the form was valid and if in preview mode the preview was approved
             if ($isFormValid && (!$this->isInPreviewMode() || $this->isPreviewApproved())) {
+                $object = $form->getData();
+                $this->admin->setSubject($object);
```

Comment:
Oh wait, [there is a clone](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Form.php#L328), so it makes sense!